book/4 - MultiWorlds/3 - Linking Worlds with a Plugin.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The easiest way to link worlds together is using a dedicated plugin, such as <a href="http://forum.mc-server.org/showthread.php?tid=2157">Portal V2</a>. More plugins can be found at the <a href="http://forum.mc-server.org/forumdisplay.php?fid=2">respository</a>.
```

Comment:
Oh, I didn't think of that.

Still though, I don't think it's the easiest way to link world's together :P